week-1/exercises/paths.js

```diff
@@ -13,6 +13,10 @@ const filePath = "/Users/mitch/cyf/Module-JS1/week-1/interpret/file.txt";
 const lastSlashIndex = filePath.lastIndexOf("/");
 const base = filePath.slice(lastSlashIndex + 1);
 console.log(`The base part of ${filePath} is ${base}`);
-
+console.log(lastSlashIndex);
 // Create a variable to store the dir part of the filePath variable
 // Create a variable to store the ext part of the variable
+let = filePath ("/")
```

Comment:
You did no create variable as the instructions said in 17, you tried but you did not create vaiarable in line 19, and also you did not solve the problem. Can you please review this please?